app/src/test/java/com/segment/analytics/android/integrations/appsflyer/AppsflyerIntegration_ConversionListenerTests.java

```diff
@@ -0,0 +1,274 @@
+package com.segment.analytics.android.integrations.appsflyer;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import android.app.Application;
+import android.content.Context;
+import android.content.SharedPreferences;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import com.segment.analytics.Analytics;
+import com.segment.analytics.Properties;
+import com.segment.analytics.ValueMap;
+import static  org.mockito.Mockito.*;
+import java.lang.reflect.Method;
+import java.util.HashMap;
+import java.util.Map;
+
+@RunWith(AndroidJUnit4.class)
+public class AppsflyerIntegration_ConversionListenerTests {
```

Comment:
Please change the class name to match the regex   `[A-Z][A-Za-z\d]*Test(s|Case)?|Test[A-Z][A-Za-z\d]*`